### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by lmacken**
• New gpg/pass + ansible vault setup for per-host credentials
• Enable ssh agent forwarding, so we can run ansible tasks inside of remote VMs
• IDS improvements
• Improvements to the nmap scanner
• Quote the backup path in the cron job
• Tag lynis, rhunter, backups, selinux, etc.
• Added a synergy role
• Update the releases in ship-it.py

### CODE REVIEW

File: roles/security/tasks/suricata.yml [hunk 1]
```diff
@@ -71,10 +69,33 @@
...
+  - ids
+
+- name: suricata | Disable specific IDS alerts
+  lineinfile: dest=/etc/suricata/rules/{{ item.rules }}.rules
+              regexp='^(alert .*{{ item.alert }}.*)$'
+              line='#\1'
+              backrefs=yes
+              backup=yes
+  with_items:
+    - { rules: 'emerging-policy', alert: 'YUM User-Agent' }
```
Comment (ralphbean) [inline roles/security/tasks/suricata.yml:89]: Can we move these rules into vars in groups/hosts?  That was.. you can disable some but I and other can still keep them active.  Thought for another PR.

**Comment by ralphbean** [comment]
Awesome!  Thanks man :)

File: roles/security/tasks/suricata.yml (hunk 1 above)
Comment (lmacken) [inline roles/security/tasks/suricata.yml:89]: Yeah, definitely, that sounds like the way to go. I was thinking about making them role arguments, but being host variables would make it cleaner :)

Comment (ralphbean) [inline roles/security/tasks/suricata.yml:89]: We could even do defaults in `roles/security/vars/` and then _optionally_ override them at the host/group level or as arguments to the role.

**Pull request merged**